src/app/runtime/runtime.wac.ts

```diff
@@ -71,28 +71,17 @@ export class WACRuntime implements Runtime {
 
     public ConnectToIISHost(): Observable<ApiConnection> {
         if (!this._connecting) {
-            let getApiKey = Observable.forkJoin(
-                this.wac.NodeName,
-                this.GetApiKey(),
-            ).map(([nodeName, apiKey], _) => {
-                var connection = new ApiConnection(nodeName)
-                if (apiKey.access_token) {
-                    connection.accessToken = apiKey.access_token
-                } else {
-                    connection.accessToken = apiKey.value
-                }
-                this._tokenId = apiKey.id
-                return connection
-            })
-
-            let ensureAccess  = Observable.forkJoin(
-                getApiKey,
-                this.powershellService.run(PowerShellScripts.admin_api_util, {
-                    command: 'ensure-permission'
-                }),
-            ).map(([key, _], __) => key)
-
-            this._connecting = ensureAccess.shareReplay()
+            this._connecting = this.wac.NodeName.mergeMap(nodeName =>

```

Comment:
My bad, forkJoin cannot guarantee execution order so the previous version is bugged, and also I end up calling `ensure-permission` twice by mistake